python/pyspark/ml/feature.py

```diff
@@ -819,6 +819,51 @@ class StringIndexerModel(JavaModel):
     """
 
 
+class StopWordsRemover(JavaTransformer, HasInputCol, HasOutputCol):
+    """
+    .. note:: Experimental
+
+    A feature transformer that filters out stop words from input.
+    Note: null values from input array are preserved unless adding null to stopWords explicitly.
+    """
+    # a placeholder to make the stopwords show up in generated doc
+    stopWords = Param(Params._dummy(), "stopWords", "The words to be filtered out")
+
+    @keyword_only
+    def __init__(self, inputCol=None, outputCol=None, stopWords=[]):
```

Comment:
Can we default `stopWords` to the same list of words as the Scala version (i.e. the words in `StopWords.EnglishStopWords`)? This may require modifying access permissions of the Scala code (but if you do this, do you mind changing `EnglishStopWords` to `ENGLISH_STOP_WORDS` since it is a constant)?